### PR TITLE
ACM-14584: stolostron/clusterlifecycle-state-metrics use ubi9

### PR DESCRIPTION
**Pull request by jacobbaungard**
Use ubi9 as the base image to match what is set in the repos Dockerfile.prow.

### CODE REVIEW

**Comment by jacobbaungard** [comment]
/pj-rehearse pull-ci-stolostron-clusterlifecycle-state-metrics-main-sonar-pre-submit

File: ci-operator/config/stolostron/clusterlifecycle-state-metrics/stolostron-clusterlifecycle-state-metrics-main.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ base_images:
   base:
     name: ubi-minimal
     namespace: ocp
-    tag: "8"
+    tag: "9"
```
Comment (haoqing0110) [inline ci-operator/config/stolostron/clusterlifecycle-state-metrics/stolostron-clusterlifecycle-state-metrics-main.yaml:5]: Should change the file .stolostron-clusterlifecycle-state-metrics-backplane-2.7.yaml as well.

**Comment by elgnay** [comment]
/pj-rehearse pull-ci-stolostron-clusterlifecycle-state-metrics-main-sonar-pre-submit

**Pull request merged**